SPECS/python-tensorboard/genereate_source_tarball.sh

```diff
@@ -1,14 +1,79 @@
 #!/bin/bash
+# Copyright (c) Microsoft Corporation.
```

Comment:
Please also update `python-tensorboard.spec` by splitting the source into the original upstream source and the additional offline dependencies. The #8942 introduced bundling of the two and that's making locating the original source harder.

Reply:
I haven't got time yet to try "bazel sync" to make this work. 
If it doesn't work, we might be stuck with this solution.